internal/metamorph/store/postgresql/postgres.go

```diff
@@ -520,10 +518,9 @@ func (p *PostgreSQL) UpdateStatusBulk(ctx context.Context, updates []store.Updat
 						UNNEST($1::BYTEA[], $2::INT[], $3::TEXT[])
 				AS t(hash, status, reject_reason)
 			) AS bulk_query
-			WHERE
-			metamorph.transactions.hash=bulk_query.hash
-				AND ( metamorph.transactions.status < bulk_query.status OR ( metamorph.transactions.status = $5 AND bulk_query.status = $4 ))
-			    AND metamorph.transactions.status != $6
+			WHERE metamorph.transactions.hash=bulk_query.hash
+				AND metamorph.transactions.status < bulk_query.status
+			  AND metamorph.transactions.status != $4
```

Comment:
Why do we need `AND metamorph.transactions.status != MINED` ? If we don't want to update these records maybe it's better to remove them from the arguments?